src/lib.rs

```diff
@@ -221,27 +150,22 @@ impl fmt::Display for ServerId {
 }
 
 /// The ID of a Raft client.
-#[derive(Copy, Clone, Hash, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq)]
```

Comment:
Is this going to print the same value?  The previous implementation prints the Display format for the UUID.